R/importWQP.R

```diff
@@ -57,8 +57,9 @@ importWQP <- function(obs_url, zip=TRUE, tz=""){
   if(!file.exists(obs_url)){
     
     if(zip){
+      message("zip encoding access still in development")
       temp <- tempfile()
-      # options(timeout = 120)
+      options(timeout = 120)
```

Comment:
120 was specifically chosen back in the day for a reason...it's not a random number.